app/models/invenio_rdm_record_converter.rb

```diff
@@ -89,13 +91,23 @@ def owner_info(generic_file)
     end
   end
 
-  def record_for_export(generic_file)
+  def file_permissions
+    permission_data = Hash.new { |h,k| h[k] = [] }
+
+    @generic_file.permissions.each do |permission|
```

Comment:
Ah, you just construct it here. I'll update my above comment, but I think in this case you should just go with `Hash.new`